.circleci/config.yml

```diff
@@ -1,14 +1,14 @@
-orbs:
-  win: circleci/windows@1.0.0
-
+version: 2
 jobs:
-  build:
-    executor: machine
+  build_and_test:
+    machine:
+      image: ubuntu-1604:201903-01
     steps:
       - checkout
-      - run: 'g++ -std=c++11 *.cpp *.h'
+      - run: 'make'
+      - run: ./projekt < test.txt > output.txt      
```

Comment:
Kulon job-ban legyen a teszteles. Illetve ha mar tud fajlba menteni, meg abbol dolgozni, akkor az is legyen tesztelve.